config/environments/test.rb

```diff
@@ -39,7 +39,4 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
-
-  config.neo4j.session_type = :server_db
-  config.neo4j.session_path = 'http://localhost:7475'
```

Comment:
I'm using another database to run the tests since I rebuild the data before running the tests. 
So I use a development database in port 7474 and another instance in 7475